utils/nft-utils/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
We have a lot of packages in the workspace already. Did we look at options that don't add this at the top level? Maybe we can make this a binary in the other nft package?

Reply:
had a discussion with team and it would be good to come up with a more generalized structure to place util libs and util bins for each module. created an issue to track this https://github.com/Sovereign-Labs/sovereign-sdk/issues/957 for now, resolving the comment and keeping the structure at `sovereign/utils/nft-utils`